lib/media-helper.js

```diff
@@ -84,9 +84,6 @@ export function writeScreenshot( data, prefix, i18nScreenshot ) {
 	}
 
 	let directoryName = 'screenshots';
-	if ( i18nScreenshot === true ) {
```

Comment:
Is there a reason you removed this?  And can we reference the envvar here as well just to make sure it's always in sync?